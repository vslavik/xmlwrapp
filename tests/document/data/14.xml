<root xmlns:xi="http://www.w3.org/2001/XInclude">
    <child>
	<xi:include href="14inc.xml"/>
    </child>
</root>
