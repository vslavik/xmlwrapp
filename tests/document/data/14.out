1
<?xml version="1.0"?>
<root xmlns:xi="http://www.w3.org/2001/XInclude">
    <child>
	<subchild xml:base="data/14inc.xml"><innerchild self="yes"/></subchild>
    </child>
</root>
